packages/devtools_app/lib/src/profiler/cpu_profile_controller.dart

```diff
@@ -15,6 +15,8 @@ import 'cpu_profile_service.dart';
 import 'cpu_profile_transformer.dart';
 
 class CpuProfilerController with SearchControllerMixin<CpuStackFrame> {
+  static const userTagNone = 'none';
```

Comment:
Comment whether the word 'none' has a magic meaning or is just a user friendly name for no tag.

Reply:
done